tools/cbmc/include/cbmc.h

```diff
@@ -1,16 +1,18 @@
-// CBMC pointer uses BITS bits for object id, remaining bits for offset
-// ProcessDHCPReplies needs 7 bits, most need at most 4
+// CBMC pointer uses BITS bits for object id, remaining bits for
```

Comment:
I'm having trouble parsing this comment. Could it please be rewritten? Maybe an example of what it means that "ProcessDHCPReplies needs 7 bits".

Reply:
I've updated this comment:

```
/* 
 * CBMC models a pointer as an object id and an offset into that
 * object.  The top bits of a pointer encode the object id and the
 * remaining bits encode the offset.  This means there is a bound on
 * the maximum offset into an object in CBMC, and hence a bound on the
 * size of objects in CBMC.
 */
```